samples/excel/85-preview-apis/events-formula-changed.yaml

```
@@ -0,0 +1,123 @@
+order: 3
+id: excel-events-formula-changed
+name: Formula change event
+description: Registers an event handler to detect changes to formulas.
+host: EXCEL
+api_set:
+    ExcelAPI: '1.13'
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup)); $("#registerFormulaChangeHandler").click(() => tryCatch(registerFormulaChangeHandler)); $("#triggerFormulaChange").click(() => tryCatch(triggerFormulaChange));
```

Let's put each of these on a single line.
```suggestion
        $("#setup").click(() => tryCatch(setup)); 
        $("#registerFormulaChangeHandler").click(() => tryCatch(registerFormulaChangeHandler)); 
        $("#triggerFormulaChange").click(() => tryCatch(triggerFormulaChange));
```

---

I'm not sure why this happened. I will keep an eye on it in the future. Thank you!

---

The Script Lab "Copy to Clipboard" omits the newlines in that section (among others). No idea why.